search-resources-api/src/main/java/no/unit/nva/search2/ExportResourceHandler.java

```diff
@@ -0,0 +1,60 @@
+package no.unit.nva.search2;
+
+import static no.unit.nva.search2.common.constant.Defaults.DEFAULT_RESPONSE_MEDIA_TYPES;
+import static no.unit.nva.search2.common.enums.PublicationStatus.PUBLISHED;
+import static no.unit.nva.search2.common.enums.PublicationStatus.PUBLISHED_METADATA;
+import static no.unit.nva.search2.resource.ResourceClient.defaultClient;
+import static no.unit.nva.search2.resource.ResourceParameter.AGGREGATION;
+import static no.unit.nva.search2.resource.ResourceParameter.FROM;
+import static no.unit.nva.search2.resource.ResourceParameter.SIZE;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.google.common.net.MediaType;
+import java.util.List;
+import no.unit.nva.search2.resource.ResourceClient;
+import no.unit.nva.search2.resource.ResourceQuery;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.presigner.S3Presigner;
+
+public class ExportResourceHandler extends ApiS3GatewayHandler<Void> {
+
+    private final ResourceClient opensearchClient;
+
+    @JacocoGenerated
+    public ExportResourceHandler() {
+        this(new Environment(),
+             defaultClient(),
+             defaultS3Client(),
+             defaultS3Presigner());
+    }
+
+    public ExportResourceHandler(Environment environment,
+                                 ResourceClient resourceClient,
+                                    S3Client s3Client,
+                                 S3Presigner s3Presigner
+    ) {
+        super(Void.class, environment, s3Client, s3Presigner);
+        this.opensearchClient = resourceClient;
+    }
+
+    @Override
+    String processS3Input(Void input, RequestInfo requestInfo, Context context) throws BadRequestException {
+        return
+            ResourceQuery.builder()
+                .fromRequestInfo(requestInfo)
+                .withRequiredParameters(FROM, SIZE, AGGREGATION)
+                .validate()
+                .build()
+                .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA)
+                .withOnlyCsvFields()
+                .doExport(opensearchClient);
```

Comment:
Since you have a separate method for export, do you need to explicit set CsvFields?